relay-config/src/config.rs

```diff
@@ -587,9 +589,9 @@ impl Default for Cache {
         Cache {
             project_expiry: 300, // 5 minutes
             project_grace_period: 0,
-            relay_expiry: 3600, // 1 hour
-            event_expiry: 600,  // 10 minutes
-            event_buffer_size: 1000,
+            relay_expiry: 3600,   // 1 hour
```

Comment:
fwiw this alignment used to be the same from L588-595

Reply:
Yeah, this is all `rustfmt` doing its thing. It aligns consecutive comments, but not all in a group.